### DISCUSSION TITLE
iOS difficult for me to use

### DISCUSSION THREAD

**Discussion by koeningerk** [community/community#75616] [Mobile]
### Select Topic Area

Product Feedback

### Body

I have difficulty using the GitHub app on my iPhone because I have trouble typing on my iPhone.

Please make the GitHub app Mac OS compliant so it will work as an IOS app on Mac OS. This should not require a special version for Macs, just added functionality to the base app.

**Comment by koeningerk** [comment]
It worked. Many thanks.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @koeningerk! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
